flocker/ca/_ca.py

```diff
@@ -95,8 +99,8 @@ def create_certificate_authority(keypair, dn, request, serial, expiry, digest):
 
     :param int serial: The certificate serial number.
 
-    :param int expiry: Number of seconds from now until this certificate
-        should expire.
+    :param str expiry: ASN1 formatted datetime string representing the
+        certificate's expiry date, i.e. "%Y%m%d%H%M%SZ"
```

Comment:
This isn't a validate expiration time, though.  It's a format string used to generate such a time.  Put an actual date into this example.